### ISSUE TITLE
Rendering math expressions / formulae

### ISSUE THREAD

**Issue by koseduhemak** [docToolchain/docToolchain#418]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when there are no formulas rendered (math).

**Describe the solution you'd like**
Integration of latexmath or better https://github.com/asciidoctor/asciidoctor-mathematical.

**Describe alternatives you've considered**
In the end they both work the same rendering text to awesome formulas (output format e.g. png or svg). asciidoctor-mathematical seems to be more active repository so I would prefer it.

**Comment by rdmueller** [comment]
Good Idea. But doesn't AsciiDoc support STEM out of the box?
https://asciidoctor.org/docs/user-manual/#activating-stem-support

and to use asciidoctor-mathematical, we would need a versin wrapped for java... let's see...

**Comment by koseduhemak** [comment]
Yes and it does work indeed for generated HTML. But it does not work for PDF and for publishing to Confluence... It would be awesome if we just can convert the formulae to svg / png files and render them as images.

**Comment by rdmueller** [comment]
ok, so I will give it a try and see why it does not render for PDF and Confluence.
Thanx for this update!

**Comment by k-werner** [comment]
An alternative way to render formulas is PlantUML. See https://plantuml.com/de/ascii-math for more informationen. 

I've tried it and it works with PDF. Unfortunately only the JLatexMath option worked for me:
`@startlatex
\sum_{i=0}^{n-1} (a_i + b_i^2)
@endlatex`

It is necessary to install the addtional jar files as decribed in the link above.

**Comment by ambition-consulting** [comment]
Does this work now @rdmueller ?

**Comment by ambition-consulting** [comment]
Scrap that, this works

```
When stem:[a != 0], there are two solutions to stem:[ax^2 + bx + c = 0] and they are

[stem]
++++
x = (-b +- sqrt(b^2-4ac))/(2a)
++++
```

**Comment by rehsack** [comment]
@ambition-consulting seems not to work for pdf, either :(
@rdmueller any progress on this topic?
